srcpkgs/nicotine+/template

```diff
@@ -1,15 +1,15 @@
 # Template file for 'nicotine+'
 pkgname=nicotine+
-version=3.2.9
-revision=2
+version=3.3.0
+revision=1
 build_style=python3-module
 hostmakedepends="python3-setuptools gettext"
-depends="gspell gtk+3 python3-gobject"
+depends="gspell gtk4 python3-gobject"
```

Comment:
gspell is no longer used (only compatible with GTK 3)

Reply:
thanks